tests/utils/verification/volumeproperties.go

```diff
@@ -188,3 +188,13 @@ func VerifyDetachedStatus(name, hostName, esxName string) bool {
 	log.Printf("Timed out to poll status\n")
 	return false
 }
+
+// GetPolicyNameForVol returns the vsan policy name used by the volume using docker cli
+func GetPolicyNameForVol(volName string, hostname string) string {
```

Comment:
In the existing code we are following a convention to put the hostname as the 1st parameter, so let's just follow this.